src/main/kotlin/org/opensearch/replication/action/changes/TransportGetChangesAction.kt

```diff
@@ -112,6 +123,23 @@ class TransportGetChangesAction @Inject constructor(threadPool: ThreadPool, clus
                         }
                     }
                 }
+
+                val tookInNanos = System.nanoTime() - relativeStartNanos
+                val tookInMillis = TimeUnit.NANOSECONDS.toMillis(tookInNanos)
+                if (fetchFromTranslog) {
+                    indexMetric.latencyTlog.addAndGet(tookInMillis)
+                    indexMetric.opsTlog.addAndGet(ops.size.toLong())
+                } else {
+                    indexMetric.latencyLucene.addAndGet(tookInMillis)
+                    indexMetric.opsLucene.addAndGet(ops.size.toLong())
+                }
```

Comment:
Actually just noticed that there is fallback possibility after attempting translog. Should we locate the code directly within the if blocks?

Reply:
Good catch. updated `relativeStartNanos` in that case.  Inclined  to keep code here so that all stats code remain at the same place.